### PR TITLE
add sccp line support

### PR DESCRIPTION
**Pull request by sboily**

### CODE REVIEW

File: wazo_ui/plugins/line/view.py [hunk 1]
```diff
@@ -26,15 +27,33 @@ def index(self):
...
+
+        try:
+            self.service.create(resources)
+        except HTTPError as error:
+            form = self._fill_form_error(form, error)
+            self._flash_http_error(error)
+            return self._new(form)
+
+        flash(l_('%(resource)s: Resource has been created', resource=self.resource), 'success')
+        return self._redirect_for('index')
```
Comment (fblackburn1) [inline wazo_ui/plugins/line/view.py:53]: From what I understand, the only modification to this method is `self._redirect_for`
What was not working with the default implementation (`self._redirect_referrer_or`) ??

Comment (sboily) [reply wazo_ui/plugins/line/view.py:53]: Because the referrer is the popup.

Comment (fblackburn1) [reply wazo_ui/plugins/line/view.py:53]: Ah true thank you

Comment (sboily) [reply wazo_ui/plugins/line/view.py:53]: This issue exist before this branch. If you prefer we could create another PR for fix it.

**Review by fblackburn1 — APPROVED** [review]

File: wazo_ui/plugins/line/view.py (hunk 1 above)
Comment (fblackburn1) [reply wazo_ui/plugins/line/view.py:53]: Nah, it's ok like this

**Pull request merged**